client/components/Signup.js

```diff
@@ -85,16 +75,10 @@ class Signup extends Component {
                             name='username'
                             floatingLabelText='Username'
                             type='input'
-                            value={this.state.account.username}
+                            value={this.state.account.username} 
                             onChange={this.handleChange}
-                            errorStyle={(this.props.message && this.props.message==='Username available') ? {color: 'green'} : {color: 'red'}}
-                            errorText={this.props.message} 
-                            hintText='(Check availability before creating account)'
-                            hintStyle={{ fontSize: '10px' }}
                         />
                         <br />
-                        <FlatButton type='button' onClick={this.usernameCheck}>Check Availability</FlatButton>
-                        <br />
                         <br />
                         <TextField
```

Comment:
remove line breaks, they're not needed- there is too much space between each field